views/admin_approvals.jade

```diff
@@ -2,24 +2,55 @@ extends layout
 
 block bodyScripts
     script(type='text/javascript').
+        function approveSubscription(action, item) {
+          $.ajax({
+            url: `/admin/approvals/${action}`,
+            type: "POST",
+            dataType: "json",
+            data: {
+              id: item.id,
+              app: item.application.id,
+              api: item.api.id
+            }
+          })
+            .done(function (data, status) {
+                const sorting = $("#approvalsGrid").jsGrid("getSorting");
```

Comment:
Because of client pagination/sorting/filtering, we should store a state of the grid before reloading